index.md

```diff
@@ -3,6 +3,10 @@ layout: page
 title: "Welcome to Portworx Docs"
 keywords: portworx, px-enterprise, px-developer, containers, storage
 sidebar: home_sidebar
+youtubeId : 0zTjOly0vkA
+redirect_from:
+  - /architecture.html
```

Comment:
This causes a build failure. Also `architecture.md` seems like a page worth keeping.

@michaelferranti should the content of this be included in the home page?